dump_post_process/convert_results_to_ellisysv2.sh

```diff
@@ -30,17 +30,17 @@ ItemFormat:LE version=1.1")
 
 }
 {
-TI = $1 * 1000;
+TI = $1 * 1000.0;
 RF_CHANNEL=rfchannel($3);
 AA=$4;
 Mod=$5;
 Packet=$10;
 RSSI=-40;
 if (Mod == 16){
-printf("Item time=%i aa=%s rssi=%i rfchannel=%i phy=1Mbps rawdata=\"%s\"\n",TI,AA,RSSI, RF_CHANNEL, rtrim(Packet));
+printf("Item time=%.0f aa=%s rssi=%i rfchannel=%i phy=1Mbps rawdata=\"%s\"\n",TI,AA,RSSI, RF_CHANNEL, rtrim(Packet));
```

Comment:
Curiosity question - Why do we print time in floats now?

Reply:
https://github.com/JDRFElectromag/bsim/pull/18
This was to fix saturation of timestamps for long data acquisitions.